pkg/interop/contract/contract.go

```diff
@@ -55,3 +55,11 @@ func CreateStandardAccount(pub interop.PublicKey) []byte {
 func GetCallFlags() int64 {
 	return 0
 }
+
+// Call executes previously deployed blockchain contract with specified hash
+// (160 bit in BE form represented as 20-byte slice) using provided arguments.
```

Comment:
Well, we have `interop.Hash160` for that now, it's no longer a 20-byte slice.

Reply:
Fixed.